### PR TITLE
ambiguous words: clarify tools words

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
ambiguous words: clarify tools words for lighting, dm, dumpling storage space

Except for the BR doc, all changes are confirmed by @sunzhaoyang.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dm/deploy-a-dm-cluster-using-tiup.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ aliases: ['/docs-cn/tidb-data-migration/dev/deploy-a-dm-cluster-using-ansible/']
 
 ## 前提条件
 
-当 DM 执行全量数据复制任务时，每个 DM-worker 只绑定一个上游数据库。DM-worker 首先在上游导出全部数据，然后将数据导入下游数据库。因此，DM-worker 的主机需要有足够的存储空间，具体存储路径在后续创建迁移任务时指定。
+当 DM 执行全量数据复制任务时，每个 DM-worker 只绑定一个上游数据库。DM-worker 首先在上游导出全部数据，然后将数据导入下游数据库。因此，DM-worker 的主机空间需要容纳所有要导出上游表，具体存储路径在后续创建迁移任务时指定。
```
Comment (Oreoxmt) [inline dm/deploy-a-dm-cluster-using-tiup.md:19]:
```suggestion
当 DM 执行全量数据复制任务时，每个 DM-worker 只绑定一个上游数据库。DM-worker 首先在上游导出全部数据，然后将数据导入下游数据库。因此，DM-worker 的主机空间需要容纳所有要导出的上游表，具体存储路径在后续创建迁移任务时指定。
```

File: br/backup-and-restore-use-cases.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ BR 可以直接将命令下发到 TiKV 集群来执行备份和恢复，不依
 运行 [`br backup`](/br/use-br-command-line-tool.md#br-命令行描述) 命令进行备份前，请确保以下条件：
 
 1. TiDB 集群中没有正在运行中的 DDL。
-2. 用于创建备份的存储设备有足够的空间。
+2. 用于创建备份的存储设备有足够的空间（根据备份集群的 1/3 的磁盘空间即可）。
```
Comment (Oreoxmt) [inline br/backup-and-restore-use-cases.md:75]:
```suggestion
2. 用于创建备份的存储设备有足够的空间（具有备份集群的 1/3 的磁盘空间即可）。
```

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -182,6 +182,8 @@ delta_index_cache_size = 0
     # TiFlash 存储引擎的压缩级别，默认为 1。如果 dt_compression_method 设置为 LZ4，推荐将该值设为 1；如果 dt_compression_method 设置为 zstd ，推荐将该值设为 -1 或 1，设置为 -1 的压缩率更小，但是读性能会更好；如果 dt_compression_method 设置为 LZ4HC，推荐将该值设为 9。
     dt_compression_level = 1
 
+    ## 从 v5.4.0 引入，表示触发 TiFlash PageFile（log 类型） 进行 hardlink 时最小的 valid rate 值，取值范围为 [0.1,1.0]。取值越小，hardlink 越频繁；取值越大，触发 hardlink 条件越苛刻。默认为 2.0，即不开启 PageFile 的 hardlink 功能。如果 GC 时间过长，建议将该值修改成 0.8 以开启 hardlink 功能。
+    dt_storage_pool_log_gc_force_hardlink_rate = 2.0
```
Comment (TomShawn) [inline tiflash/tiflash-configuration.md:186]:
```suggestion
```

File: br/backup-and-restore-use-cases.md [hunk 4]
```diff
@@ -72,7 +72,7 @@ BR 可以直接将命令下发到 TiKV 集群来执行备份和恢复，不依
 运行 [`br backup`](/br/use-br-command-line-tool.md#br-命令行描述) 命令进行备份前，请确保以下条件：
 
 1. TiDB 集群中没有正在运行中的 DDL。
-2. 用于创建备份的存储设备有足够的空间。
+2. 用于创建备份的存储设备有足够的空间（具有备份集群的 1/3 的磁盘空间即可）。
```
Comment (TomShawn) [inline br/backup-and-restore-use-cases.md:75]: confirmed with @3pointer

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Review by sunzhaoyang — APPROVED** [review]
LGTM

**Pull request merged**